<?xml version="1.0"?>

<launch>
  <node name="sync_sensors_node" pkg="sync_sensors" type="sync_sensors_node" output="screen">
    <param name="use_3d"                  value="0"/>
    <param name="is_test"                 value="1"/>

    <param name="topic_cam"               value="/image"/>
    <param name="topic_2d"                value="/laser_scan"/>
    <param name="topic_3d"                value="/velodyne_points"/>
  </node>
</launch>